globals.go

```diff
@@ -298,6 +298,19 @@ var (
 		HostProvisionFailed,
 	}
 
+	// Set of host status values that can be user set via the API
+	ValidUserSetStatus = []string{
+		HostRunning,
+		HostTerminated,
+		HostUninitialized,
```

Comment:
omit lines 305-309.

it's arguable that HostTerminated shouldn't be allowed either.